api/maven-api-core/src/main/java/org/apache/maven/api/SourceRoot.java

```diff
@@ -62,8 +72,11 @@ default ProjectScope scope() {
 
     /**
      * {@return the language of the source files}.
+     * The default value is {@link Language#JAVA_FAMILY}.
```

Comment:
Just use the constant value here. No need to hide this behind a variable in the docs.